docs/content/using/changing_host_connectivity.md

```diff
@@ -0,0 +1,16 @@
+# Changing host connectivity
```

Comment:
@matancarmeli7 how is this different than `Recovering from an Error state`?

Reply:
Changing host connectivity has a step in between the `kubectl get hostdefinition` and Redeploy steps.
("From the host, change the host connectivity type.")

Reply:
@matancarmeli7  please confirm

Reply:
IMO recovering from error could include changing node connectivity as well

Reply:
@matancarmeli7  can this include changing node connectivity? if so i can add it as an optional step. 
again, these are different use cases so they should be kept separately.

Reply:
AFAIK they could easily (perhaps even usually) be the same use case

Reply:
the only difference between them is that the  `Changing host connectivity` has another step before the redeploy, we can say that the process is the same as `Recovering from an Error state` and the user should do another step before the redeploy and explain about this step, but I don't know what is the right way handle those type of cases